[32mINFO    [0m root:test_04_user.py:37 ==========开始执行用例：user_group_add_02==========
[32mINFO    [0m root:runcase.py:31 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
[32mINFO    [0m root:handleyaml.py:94 Yaml测试用例基础架构检查通过
[32mINFO    [0m root:runcase.py:50 重组后的用例信息：{'name': '新增用户组-必填字段校验', 'description': '缺少父用户组id字段，接口请求失败，提示信息正确', 'request': {'data': {'groupName': '自动化测试根组'}, 'method': 'POST', 'address': '/api/user/v1/group', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/group'}}, 'validate': {'code': 200, 'response': {'statusCode': 198145, 'msg': '父用户组不能为空'}}, 'epic': '用户管理', 'feature': '基础数据管理-用户与机构管理', 'story': '新增用户组接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'user_group_add_02'}
[32mINFO    [0m root:runcase.py:81 请求地址：https://192.168.148.174:31000/api/user/v1/group
[32mINFO    [0m root:runcase.py:82 请求方法：POST
[32mINFO    [0m root:runcase.py:83 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/group'}
[32mINFO    [0m root:runcase.py:93 请求参数：{'groupName': '自动化测试根组'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "POST /api/user/v1/group HTTP/1.1" 200 54
[32mINFO    [0m root:runcase.py:155 请求接口结果： {'statusCode': 198145, 'msg': '父用户组不能为空'}
[32mINFO    [0m root:checkresult.py:152 code断言通过, 期望状态码:200, 实际状态码:200
[32mINFO    [0m root:checkresult.py:87 返回结果断言通过, 期望返回值:{'statusCode': 198145, 'msg': '父用户组不能为空'}, 实际返回值:{'statusCode': 198145, 'msg': '父用户组不能为空'}
[32mINFO    [0m root:runcase.py:61 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-